Editor/Resources/EditorWindow/Components/DeploymentScenarios.uxml

```diff
@@ -0,0 +1,41 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <Style src="../common.uss"/>
+    <ui:GroupBox class="separator">
+        <ui:VisualElement class="form-row form-row--free">
+            <ui:Label name="ManagedEC2ScenarioSingleFleetLabel" class="form-row__label" text="Single-Region Fleet"/>
+            <ui:VisualElement class="form-row__input">
+                <ui:RadioButton name="ManagedEC2ScenarioSingleFleetRadio" class="form-row__radio-button" text="Creates a game backend service with a single Amazon GameLift fleet."/>
+                <ui:VisualElement class="external-link">
+                    <ui:Label name="ManagedEC2ScenarioSingleFleetLink" class="form-row__text-display input-hint foldout__link" text="Learn More"/>
+                    <ui:Image name="ExternalLink" class="icon"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement name="FleetTypeRadioButton2Group" class="form-row form-row--free">
```

Comment:
Please rename these to the deployment scenario similar to the names of the child elements. This isn't the most readable.

Reply:
renamed